zmq/backend/cython/checkrc.pxd

```diff
@@ -1,7 +1,12 @@
+from libc.stdio cimport printf
```

Comment:
I've never worked with cython before, but I don't think this is being used?  Probably a left over from some debugging you did?